revenue_app/tests.py

```diff
@@ -195,10 +189,11 @@ def test_group_transactions(self, by, expected_length):
         ({'event_id': '35210860'}, 5),
         ({'event_id': '88128252'}, 7),
         ({'invalid_key': 'something'}, 27),
-        ({'eventholder_user_id': '', 'organizer_id': '', 'start_date': '', 'end_date': ''}, 27),
+        ({'eventholder_user_id': '', 'start_date': '', 'end_date': ''}, 27),
+        ({'eventholder_user_id': '696421958'}, 6),
         ({'eventholder_user_id': '696421958', 'start_date': '2018-08-06'}, 1),
         ({'eventholder_user_id': '696421958', 'start_date': '2018-08-06', 'invalid_key': 'something'}, 1),
-        ({'organizer_id': '696421958', 'start_date': '2018-08-06', 'end_date': '2018-08-10'}, 5),
+        ({'eventholder_user_id': '696421958', 'start_date': '2018-08-07', 'end_date': '2018-08-09'}, 3),
```

Comment:
Why? Tests should not depend on the dataset you're working with atm

Reply:
We should never use 'organizer_id'. Datasets always have 'eventholder_user_id', and what we were doing before was accepting an organizer_id and use it as an eventholder_user_id. Something like `transactions['eventholder_user_id'] == organizer_id`

Reply:
I;m sorry, what I don't understand is you having to change the dates (not the key, which I do understand)

Reply:
Oh! Because we changed the examples dataset to include more information, and to make it easier for us we ended changing the dates (and some other values). We could have changed it without changing the dates, but it was more work

Reply:
Can you point me to this example dataset? I'm not seeing it in this PR, which seems weird.

Reply:
https://github.com/evbeda/revenue_latam/tree/filter-by-currency/revenue_app/tests Here are both datasets